### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by AnuragSahu11**
*Made Navbar responsive
*Completed playlist logic
*Added Single video page
*Added dynamic routes for playlist, single video page
( please ignore styling, got to work on it )

### CODE REVIEW

File: TradePlay/src/components/modals/add-to-playlist-small-modal.jsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { usePlaylist } from "../../context/playlist-context";
+
+const AddToPlaylistSmallModal = ({ videoId, switchModal }) => {
+  const { playlistState, playlistDispatch } = usePlaylist();
+  const clickHandler = ({ name }) => {
```
Comment (shuklahimanshu81) [inline TradePlay/src/components/modals/add-to-playlist-small-modal.jsx:5]: Try naming the function with a meaningful name.

File: TradePlay/src/components/navbar/navbar.jsx [hunk 2]
```diff
@@ -1,15 +1,34 @@
...
+      {/* <div className="nav-brand">
         <a href="" className="logo-a">
           <i className="fas is-primary fa-play"></i>
         </a>
       </div>
       <div className="nav-hamburger">
-        <i className="fas m-r-2 fas fa-moon is-black" />
         <i className="fas fa-bars" />
-      </div>
+      </div> */}
```
Comment (shuklahimanshu81) [inline TradePlay/src/components/navbar/navbar.jsx:31]: Remove these comments

File: TradePlay/src/backend/db/videos.js [hunk 3]
```diff
@@ -5,39 +5,82 @@
...
+  //   title: "Neural Networks from Scratch - P.1 Intro and Neuron Code",
+  //   creator: "Sentdex",
+  //   videoLink: "",
+  //   description:
+  //     "At vero eos et accusamus et iusto odio dignissimos ducimus qui blanditiis praesentium voluptatum deleniti atque corrupti quos dolores et quas molestias excepturi sint occaecati cupiditate non provident, similique sunt in culpa qui officia deserunt mollitia animi, id est laborum et dolorum fuga.",
+  //   views: 300,
+  //   category: "stock",
+  // },
+  // {
+  //   _id: "mindset_1",
```
Comment (shuklahimanshu81) [inline TradePlay/src/backend/db/videos.js:77]: remove these comments

**Review by shuklahimanshu81 — APPROVED** [review]
Just remove the unnecessary comments and if you can also change the name as suggested. Other than that code structure is clean. One thing that I liked most was the way you used comments in watchLaterController to explain the working of handlers . I will also try to implement them in my code.

File: TradePlay/src/components/modals/add-to-playlist-small-modal.jsx (hunk 1 above)
Comment (AnuragSahu11) [reply TradePlay/src/components/modals/add-to-playlist-small-modal.jsx:5]: Yes I will keep that on mind from now on

File: TradePlay/src/components/navbar/navbar.jsx (hunk 2 above)
Comment (AnuragSahu11) [reply TradePlay/src/components/navbar/navbar.jsx:31]: yes removed

File: TradePlay/src/backend/db/videos.js (hunk 3 above)
Comment (AnuragSahu11) [reply TradePlay/src/backend/db/videos.js:77]: yes

**Review by junaidshaikh-js — APPROVED** [review]
Good work! 

Make changes suggested by the fellow reviewer. Everything else looks good. Can be merged.

**Pull request merged**